schedule-pod-restart/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for azure-pipeline-environment.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
```

Comment:
Some pieces of the documented configuration is not here, is that on purpose?

https://github.com/CoditEU/charts/pull/20/files?file-filters%5B%5D=.html&file-filters%5B%5D=.md&file-filters%5B%5D=.tgz&file-filters%5B%5D=.tpl&file-filters%5B%5D=.yaml&file-filters%5B%5D=.yml&file-filters%5B%5D=dotfile#diff-034c0482a68bfdaaeb21123826fb3f5d80fffa1373549cf3da886248cc4cc3e4R34

Reply:
Added missing values in commit https://github.com/CoditEU/charts/pull/20/commits/807d90c57c6f579bacdf4b908af7e96a4a3a2d35